src/app/combo/combo.sample.ts

```diff
@@ -64,6 +70,8 @@ export class ComboSampleComponent implements OnInit, AfterViewInit {
     public items: any[] = [];
     public values1: Array<any> = ['Arizona'];
     public singleValue = 'Arizona';
+    public selectedValueSimpleComboEvent = 'Connecticut';
```

Comment:
We already have the `singleValue` member and I don't see the problem to bind multiple combos to the same member.